### ISSUE TITLE
Aliases / description search for themes

### ISSUE THREAD

**Issue by thibaultmol** [pietervdvn/MapComplete#1385]
Even though I know the theme names, I still find myself sometimes searching the wrong thing to find the theme I'm looking for.

I'll search for "Library" when trying to get the "open bookcase map"
... "charge" ... "charging stations"
... "bin" ... "waste basket" (and  "Waste")

And something to consider: if you search for "charging", should cyclofix also show up? because that also has charging stations layers.

**Comment by thibaultmol** [comment]
@pietervdvn is it supposed to know search in the description? Doesn't seem to be working?

**Comment by pietervdvn** [comment]
Yes, it does. However, caching is quite agrressive and you probably have a cached version. Can you verify in a private window?

**Comment by thibaultmol** [comment]
I tried inprivate, also doesn't work

**Comment by thibaultmol** [comment]
Is this supposed to work?
![Uploading Screenshot_20230425_160900_Chrome.jpg…]()

**Comment by thibaultmol** [comment]
Is this supposed to work?
![Screenshot_20230425_160900_Chrome](https://user-images.githubusercontent.com/1010226/234303610-21104fbe-9075-4ca4-85f9-fe112c09c029.jpg)

**Comment by thibaultmol** [comment]
AED is also a common thing I've tried to type before, but nothing shows up because it's only expecting 'defibrillators'

**Comment by pietervdvn** [comment]
Searching for 'charging stations' doesn't work right now
